index.html

```diff
@@ -0,0 +1,352 @@
+              <!-- <img
+                onclick="off()"
+                class="style-li"
+                src="/images/close-Icon.png"
+                alt="close-Icon"
+              /> -->
+              <i
+                onclick="off()"
+                class="material-icons"
+                style="font-size: 50px; color: rgb(226, 215, 215);"
+              >
+                close
+              </i>
+            </div>
+            <li onclick="off()" class="nav-item">
+              <a class="nav-link li-nav" href="./index.html">Home</a>
+            </li>
+            <li onclick="off()" class="nav-item">
+              <a class="nav-link li-nav" href="./pages/about.html">About</a>
+            </li>
+            <li onclick="off()" class="nav-item">
+              <a class="nav-link li-nav" href="#spearkers">Speakers</a>
+            </li>
+            <li onclick="off()" class="nav-item">
+              <a class="nav-link li-nav" href="#partners">Partners</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+
+    <!-- // Main Section // -->
+
+    <div id="main1" class="container-fluid">
+      <div class="container py-5 px-1">
+        <h2 class="mainText">"Hello, Sharing World"</h2>
+        <h1 class="mainText">
+          Enhance Your Skills
+          <br />
+          GLOBAL SUMMIT
+          <br />
+          2022
+        </h1>
+      </div>
+
+      <div class="container mainPara text-black my-3">
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit ad
+          laudantium voluptatum, modi natus perspiciatis. Lorem ipsum dolor sit
+          amet consectetur adipisicing elit.
+        </p>
+      </div>
+      <div class="container mt-3">
+        <h1>
+          2022.10.20(THU) ~ 21(FRI)
+        </h1>
+        <h3 style="padding-bottom: 10px; margin: 0;">
+          @ Lorem ipsum dolor sit amet consectetur adipisicing elit.
+        </h3>
+      </div>
+    </div>
+
+    <!-- // Program Section // -->
+
+    <div id="main2" class="container-fluid">
+      <div class="container text-white p-3 mb-5">
+        <h3 id="prgrmHearder">
+          Main Program
+        </h3>
+        <hr id="prgrmHr" />
+      </div>
+      <div class="row">
+        <div id="prgrmCol1" class="col prgrmCol">
+          <div class="card p-3 prgrmCard">
+            <div class="card-img">
+              <i
+                class="material-icons"
+                style="font-size: 50px; color: rgb(226, 215, 215);"
+              >
+                perm_camera_mic
+              </i>
+            </div>
+            <div>
+              <h4 class="card-title prgrmTitle">monolog</h4>
+            </div>
+            <div class="card-body">
+              <p class="card-text" style="font-size: small;">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Cum,
+                ipsa?
+              </p>
+            </div>
+          </div>
+        </div>
```

Comment:
- [ ] The main program section is made up of 5 articles and as such each article should be wrapped in the `article` tag.
- [ ] Also CSS styling should be applied in the a seperate CSS file as this is the industry standard, kindly do so for the whole project.

Reply:
Thanks for the suggestion. I used bootstrap with CSS styling